plugins/wpe-headless/includes/settings/views/headless-settings.php

```diff
@@ -41,6 +59,13 @@
 							<p class="error-message"></p>
 						<?php endif; ?>
 					</section>
+					<?php if ( ! get_option( 'permalink_structure' ) ) : ?>
+					<section>
+						<h4><?php esc_html_e( 'Enable Permalinks', 'wpe-headless' ); ?></h4>
+						<p><?php esc_html_e( 'Headless is easier with pretty permalinks! If you are unsure of which option to choose, go with "Post Name".', 'wpe-headless' ); ?></p>
```

Comment:
THe issue with"Post name" is it's not what the templates expect and it'd share a template with pages. The latter might be okay but this recommendation should probably align with our starters....maybe?

Reply:
After discussion, @moonmeister and I believe this to be a non-issue.